code/modules/events/blizzard.dm

```diff
@@ -17,8 +19,12 @@ var/blizzard_cooldown = 5 MINUTES
 			blizzard_ready = FALSE
 			command_alert(/datum/command_alert/blizzard_start)
 			W.timeleft = round(rand(2 MINUTES, 4 MINUTES),SS_WAIT_WEATHER)
-			W.next_weather = list(/datum/weather/snow/blizzard = 100)
+			// Temporarily override transitions to force blizzard
```

Comment:
Not entirely sure this is the best implementation. Maybe a proc `force_blizzard` at the climate datum level ? This way it can be tweaked rather than being bits of code into the blizzard event proc. Just a thought.

Reply:
Good idea; I think a general "force_weather" proc would be very useful. Will implement